ownCloudAppShared/Client/User Interface/ClientItemCell.swift

```diff
@@ -19,6 +19,29 @@
 import UIKit
 import ownCloudSDK
 
+extension NSMutableAttributedString {
+	var boldFont:UIFont { return UIFont.systemFont(ofSize: 17.0, weight: .semibold) }
```

Comment:
@hosy I think would be nicer to use `UIFont.preferredFont(forTextStyle:)` instead of absolute point based sizes

Reply:
good idea, changed it!